src/cbmc/cbmc_parse_options.h

```diff
@@ -46,7 +46,8 @@ class optionst;
   OPT_GOTO_CHECK \
   "(no-assertions)(no-assumptions)" \
   "(xml-ui)(xml-interface)(json-ui)" \
-  "(smt1)(smt2)(fpa)(cvc3)(cvc4)(boolector)(yices)(z3)(opensmt)(mathsat)" \
+  "(smt1)(smt2)(fpa)" \
+  "(cprover-smt2)(cvc3)(cvc4)(boolector)(yices)(z3)(mathsat)" \
```

Comment:
OpenSMT is still being developed by Hana and Natasha's teams; do we really want to remove support?

Reply:
Yes; it does not support any theories that we use.

Reply:
OK.